recipes/resiprocate/all/test_package/test_package.cpp

```diff
@@ -1,30 +1,50 @@
 #include <resip/stack/SipMessage.hxx>
 #include <resip/stack/Helper.hxx>
 #include <rutil/Logger.hxx>
-#include <fstream>
 #include <iostream>
 
 #define RESIPROCATE_SUBSYSTEM Subsystem::TEST
 
 using namespace resip;
 
+std::string sipRawData = "INVITE sip:nikolia@example.ru SIP/2.0\r\n"
```

Comment:
In case you did know about it, https://en.cppreference.com/w/cpp/language/string_literal number 6 is a "raw string literal" might have made this a little easier

Reply:
Yep, I know about raw string letteral, but as for me it would me much easier to predict line ending style in case of line concatenation.

Reply:
I just noticed the SDP content, that needs to be `\r\n` and the C++ ISO says raw strings are `\n` only  so sport on ! xD